lib/Doctrine/ORM/Internal/Hydration/AbstractHydrator.php

```diff
@@ -697,7 +697,7 @@ protected function registerManaged(ClassMetadata $class, $entity, array $data)
      *
      * @return BackedEnum|array<BackedEnum>
      */
-    private function buildEnum($value, string $enumType)
+    protected function buildEnum($value, string $enumType)
```

Comment:
If we are going to expose this to other classes, let's at least make it final
```suggestion
    final protected function buildEnum($value, string $enumType)
```

Reply:
Done